frame/fast-unstake/src/lib.rs

```diff
@@ -125,6 +123,21 @@ pub mod pallet {
 		/// The origin that can control this pallet.
 		type ControlOrigin: frame_support::traits::EnsureOrigin<Self::RuntimeOrigin>;
 
+		/// Just the `Currency::Balance` type; we have this item to allow us to constrain it to
+		/// `From<u64>`.
+		type CurrencyBalance: sp_runtime::traits::AtLeast32BitUnsigned
```

Comment:
why is this needed?

Reply:
We actually don't need this anymore afaik, this is just leftover junk I ported over from Staking setup. Going to remove